packages/beacon-node/src/chain/opPools/syncCommitteeMessagePool.ts

```diff
@@ -146,7 +146,7 @@ function aggregateSignatureInto(
  * Format `signature` into an efficient `contribution` to add more signatures in with aggregateSignatureInto()
  */
 function signatureToAggregate(
-  subnet: number,
+  subnet: Subnet,
```

Comment:
Should this be SubnetID ?

Reply:
so the CL spec only uses the `SubnetID` for attestation and blob sidecar subnets, for sync committee we have `SubcommitteeIndex` (aliased to `Subnet` above)  which is equal the subnet id